resources/web/map/js/trajectory_raw.js

```diff
@@ -77,16 +79,38 @@ function setupLayers() {
             }
         });
     }
+    if (map.getSource(`prot`) != null) return;
+    map.addSource(`prot`, {
+        type: 'geojson',
+        cluster: false,
+        data: {
+            type: 'FeatureCollection',
+            features: []
+        }
+    })
+    map.addLayer({
+        id: `prot`,
+        type: 'fill',
+        source: `prot`,
+        paint: {
+            'fill-color': "#00FF00",
```

Comment:
00ff00 is also same colour as id=5 mode for line points. (but opacity is a bit higher)

Reply:
@dgburr any thoughts on the colouring of this 🤔 (ok to have green for both?)

Reply:
ahh, hmmm then we'll have to figure some different ways to distinguish accuracy points and prot lvl 🤔 - 
toolbox currently uses colours to encode different points
basically took the colour scheme 

<img width="79" alt="image" src="https://user-images.githubusercontent.com/35755741/225160138-f65301b9-0fde-498e-9014-c5fa57676399.png">

and encoded into the accuracy points which `RTK Fixed` also shares green colour as well

Reply:
> @adrian-kong: are you asking whether PL circles and EHPE circles can have the same colour, or are you asking whether PL circles and their connecting lines can have the same colour?
> 
> IMHO it should work the same way as the Drongo defaults:
> 
> 1. Estimated accuracy ellipses are orange
> 2. Black lines connect estimated accuracy circles
> 3. PL ellipses are green

I think we need to decide on a different color scheme, part of the new console work was to change the color scheme that the console uses to line up with other tools: https://swift-nav.atlassian.net/browse/CPP-449

Reply:
<img width="339" alt="image" src="https://user-images.githubusercontent.com/35755741/226759735-282f8c85-e7b8-4859-b23b-911c01bda29c.png">

Here's the new colour scheme just to avoid drastic changes,
I've added outlines to accuracy circle and different shade of green.